homeassistant/components/light/intent.py

```diff
@@ -30,11 +30,13 @@ async def async_setup_intents(hass: HomeAssistant) -> None:
             optional_slots={
                 ("color", ATTR_RGB_COLOR): color_util.color_name_to_rgb,
                 ("temperature", ATTR_COLOR_TEMP_KELVIN): cv.positive_int,
-                ("brightness", ATTR_BRIGHTNESS_PCT): vol.All(
-                    vol.Coerce(int), vol.Range(0, 100)
-                ),
+                (
+                    "brightness",
+                    ATTR_BRIGHTNESS_PCT,
+                    "The brightness percentage of the light between 0 and 100, where 0 is off and 100 is fully lit",
```

Comment:
For data entry flow, we abuse `description` to inject a dictionary that can contain multiple keys. For future proofing, should we do that here too?